model/schedulesmodel.py

```diff
@@ -8,6 +8,7 @@
 class SchedulesModel(ndb.Model):
     """Stores scheduling data."""
     Name = ndb.StringProperty(indexed=True, required=True)
+    DisplayName = ndb.StringProperty(indexed=True, required=False)
```

Comment:
Not sure if the display name needs to be indexed. Are we looking up entities by their display name anywhere? I think only by name, right?

Reply:
See above, it seems to be mandatory for strings now.